network/middleware.go

```diff
@@ -67,6 +69,9 @@ type Overlay interface {
 	// Identities returns a list of all Flow identities on the network
 	Identities() flow.IdentityList
 
+	// GetIdentity returns the Identity associated with the given peer ID, if it exists
+	GetIdentity(peer.ID) (*flow.Identity, bool)
```

Comment:
```suggestion
	Identity(peer.ID) (*flow.Identity, bool)
```
as per https://golang.org/doc/effective_go#Getters